main.go

```diff
@@ -75,10 +76,14 @@ func sourceFromArg(arg string) (*source, error) {
 	}
 
 	// a GitHub or GitLab URL (even without the protocol):
+	var dnsErr *net.DNSError
 	src, err := readmeURL(arg)
 	if src != nil && err == nil {
 		// if there's an error, try next methods...
```

Comment:
I'm doubtful about this comment

there is no error, so why talking about trying the other one ?

It worked, no ?